Note on sustained payloads at 2MB limit (#4712)
temporalio/documentation: docs/evaluate/temporal-cloud/limits.mdx

The [BlobSizeLimitError guide](/troubleshooting/blob-size-limit-error) provides solutions for handling large payloads.

:::note

The 2 MB limit is the maximum size for a single request, not a sustained-throughput target.

> Sustained, high-volume use of large payloads can degrade Namespace performance, and Temporal reserves the right to rate limit such traffic to protect the service.

Reviewer: ```suggestion
Sustained, high-volume use of large payloads can degrade Namespace performance, and we may rate limit such traffic to protect the service.
```
I personally feel like "Temporal reserves the right to" sounds a bit aggressive, kind of like we are about to sue them. Making this a little more direct actually sounds softer?
